### ISSUE TITLE
Virtual Heightfield Mesh as possible replacement for displacement

### ISSUE THREAD

**Issue by delebash** [GameDev4K/unreal-openland#34]
Someone mentioned on unreal sensei's channel that this might be what is going to replace displacement in ue5, not much info available but something interesting to consider.  See https://www.youtube.com/watch?v=3Zl1P96oYls

**Comment by arunoda** [comment]
Yes, I saw that too. That was also speculated on the UE forums as well. We should try that out later on.
